collections/_version2-english/9-fork/4-migration-of-the-objects.md

```diff
@@ -0,0 +1,78 @@
+
+# Overview
+The following figure shows where the objects in the [parent universe]({{glossary_parent_universe}}) are migrated after a fork. (Click to enlarge)
+{% capture image_src %}{{ "/" | absolute_url }}assets/images/{{page.collection}}/fork/migration-of-the-objects/outline.svg{% endcapture %}
+{% include zoom-image.html src=image_src caption="Figure 2. overview of migration" %}
+
+The red objects can only be migrated to the [winning universe]({{glossary_winning_universe}}), the blues can only be migrated to the [losing universe]({{glossary_losing_universe}}), the purples can be migrated to one of the [child universes]({{glossary_child_universe}}), and the greens cannot be migrated to any child universe.
+
+Before a [fork]({{glossary_fork}}), all objects are in the parent universe. After a fork, in the parent universe, there are the [finalized market]({{glossary_finalized_market}}), [dispute window]({{glossary_dispute_window}}), and the objects which are attached to them. The winning universe and the losing universe are created, and the [forking market]({{glossary_forking_market}}), [non-finalized market]({{glossary_non_finalized_market}}), and [REP]({{glossary_reputation_token}}) are migrated to them. 
+
+Those can be summarized as follows:
+
+  - The migratable objects are:
+    - Forking market
+    - Non-Forking Market
+    - REP
+  - The un-migratable objects are:
+    - Finalized Market
+    - Dispute Window 
+  - The winning universe receives:
+    - Forking market and its attached objects
+    - Non-Forking market
+    - REP
+  - The losing universe receives:
+    - Forking market
+    - REP
+
+# Who creates child universes?
+As mentioned above, some objects in the [parent universe]({{glossary_parent_universe}}) can be migrated to a [child universes]({{glossary_child_universe}}), but who creates the child universe? Is it created by Augur [contracts]({{glossary_smart_contract}}) automatically? No, it is created by [REP]({{glossary_reputation_token}}) holder. The REP holder who *first* migrates their REP to the child universe creates the child universe. When REP in the parent universe first migrated to a child universe, the child universe is created. A child universe is not created until someone migrates their REP to the child universe. Even if the [forking market]({{glossary_forking_market}}) has [finalized]({{glossary_finalized_market}}) or the [forking period]({{glossary_forking_period}}) ends, the child universe is not created without migrating REP.
```

Comment:
```suggestion
As mentioned above, some objects in the [parent universe]({{glossary_parent_universe}}) can be migrated to a [child universes]({{glossary_child_universe}}), but who creates the child universe? Is it created by the Augur [contracts]({{glossary_smart_contract}}) automatically? No, it is created by [REP]({{glossary_reputation_token}}) holders. The REP holder who *first* migrates their REP to the child universe creates the child universe. When REP in the parent universe first migrated to a child universe, the child universe is created. A child universe is not created until someone migrates their REP to the child universe. Even if the [forking market]({{glossary_forking_market}}) has [finalized]({{glossary_finalized_market}}) or the [forking period]({{glossary_forking_period}}) ends, the child universe is not created without migrating REP.
```
You repeat yourself several times in this paragraph.  It may be better to simplify it down to just a simple one-sentence answer.

Reply:
> You repeat yourself several times in this paragraph. It may be better to simplify it down to just a simple one-sentence answer.

I'm going to submit this suggestion as an issue.

Reply:
I submitted an issue https://github.com/msagansk/Augur.Guide/issues/52